developers/level2.md

```diff
@@ -10,8 +10,8 @@
 
 ### Is expected to
 
-* Lean into challenges when they offer the potential to create large impact
-* Help grow the engineering team through involvement in the technology community (conferences, Socialbakers, blog posts, open source projects, etc) and referrals
+* Dive into challenges when they offer the potential to create a large impact
```

Comment:
I'd still rather use Lean

Reply:
I think that Dive into makes more sense:

Lean into = to push into or press against someone or something
Dive into = to occupy oneself suddenly and enthusiastically with something

Reply:
Lean into makes more sense in this context:   

"Dive" implies we want them to drop everything at the sight of a challenge, while "Lean" suggests that if you see an opportunity to do good you should support it

Reply:
fixed